packages/sls-env/src/index.ts

```diff
@@ -1,5 +1,7 @@
+import _get from 'lodash.get'
```

Comment:
I don't think we need a lodash dependency for get as we can use optional syntax.

`event?.property1?.property2`